Client/Info.plist

```diff
@@ -89,7 +89,7 @@
 	<string>Websites you visit may request your location.</string>
 	<key>NSMicrophoneUsageDescription</key>
 	<string>This lets you take and upload videos.</string>
-	<key>NSPhotoLibraryAddUsageDescription</key>
+	<key>NSPhotoLibraryUsageDescription</key>
```

Comment:
We cannot use this privacy entry as per default browser requirements: https://developer.apple.com/documentation/xcode/allowing_apps_and_websites_to_link_to_your_content/preparing_your_app_to_be_the_default_browser_or_email_client

Reply:
The `PHPhotoLibrary` require `NSPhotoLibraryUsageDescription` instead of `NSPhotoLibraryAddUsageDescription` plist key and we cant use it according to the default Browser criteria you mentioned.
Reverting plist change and PHPhotoLibrary Asset request in the light of this.